packages/components/src/SubHeaderBar/SubHeaderBar.component.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Action } from '../Actions';
+import ActionBar from '../ActionBar';
+import InputTitleSubHeader from './InputTitleSubHeader';
+import theme from './SubHeaderBar.scss';
+
+function SubHeaderBarActions({ components, left, right, center, className }) {
+	return (
+		<div className={className}>
+			{components.map((component, index) => (
+				<ActionBar.Content
+					key={index}
+					tag={component.tag}
+					left={left}
+					center={center}
+					right={right}
+				>
+					{component.injectedComponent}
+				</ActionBar.Content>
+			))}
+		</div>
+	);
+}
+
+SubHeaderBarActions.propTypes = {
+	components: PropTypes.array.isRequired,
+	right: PropTypes.bool,
+	center: PropTypes.bool,
+	left: PropTypes.bool,
+	className: PropTypes.string,
+};
+
+function SubHeaderBar({ onGoBack, componentsCenter, componentsRight, className, ...rest }) {
+	const defaultComponentsLeft = [
```

Comment:
Instead to use an array of object with an Element, you can use an array of Element

Reply:
All the components array (left, center, right) are used in SubHeaderBarActions. In the map func in it, we used the injectedComponent and tag (optional attr to assign css classes in the ActionBar) attr for each entry. So I need to keep the same structure.

Reply:
I'll add a new example in storybook to demonstrate the tag option.